### PR TITLE
[DASH] Update dash flow and dash tunnel

### PR DESCRIPTION
**Pull request by jimmyzhai**
1. Update SAI of dash flow to align with its p4 table in dash-pipeline
2. Update SAI of dash tunnel to align with its p4 table in dash-pipeline

### CODE REVIEW

File: experimental/saitypesextensions.h [hunk 1]
```diff
@@ -180,10 +184,32 @@ typedef enum _sai_dash_flow_enabled_key_t
...
+
+    SAI_DASH_FLOW_ACTION_DNAT = 32,
+
+    SAI_DASH_FLOW_ACTION_NAT46 = 64,
+
+    SAI_DASH_FLOW_ACTION_NAT64 = 128,
+
+    SAI_DASH_FLOW_ACTION_SNAT_PORT = 256,
+
+    SAI_DASH_FLOW_ACTION_DNAT_PORT = 512,
```
Comment (kcudnik) [inline experimental/saitypesextensions.h:212]: mayeb use arrows << like other to easy spot which bit is turned on?

File: inc/saiobject.h [hunk 2]
```diff
@@ -37,6 +37,8 @@
 #include <saisrv6.h>
 
 /* new experimental object type includes */
+#include <saiexperimentaldashtunnel.h>
+#include <saiexperimentaldashha.h>
```
Comment (kcudnik) [inline inc/saiobject.h:41]: no need for this

File: experimental/saitypesextensions.h (hunk 1 above)
Comment (zhixiongniu) [reply experimental/saitypesextensions.h:212]: agree

**Review by zhixiongniu — APPROVED** [review]

File: experimental/saitypesextensions.h (hunk 1 above)
Comment (jimmyzhai) [reply experimental/saitypesextensions.h]: fixed

File: inc/saiobject.h (hunk 2 above)
Comment (jimmyzhai) [reply inc/saiobject.h:41]: Why not? Here the list has included `saiexperimentaldashappliance.h, saiexperimentaldashflow.h, saiexperimentaldashmeter.h, saiexperimentaldashvip.h`, etc. The list is automatically updated while `make sai-headers` in DASH to have header files of all dash objects.

Comment (kcudnik) [reply inc/saiobject.h:41]: since no item is used from included files, so including them is redundant

**Review by kcudnik — APPROVED** [review]

**Pull request merged**